dask_cuda/local_cuda_cluster.py

```diff
@@ -35,7 +36,7 @@ def __init__(
         threads_per_worker=1,
         processes=True,
         memory_limit=None,
-        **kwargs
+        **kwargs,
```

Comment:
```suggestion
        **kwargs
```

Reply:
This is an issue with black.  It's fairly insistent about this, which is a bit of a pain for python 3.5 users.

Reply:
See https://github.com/python/black/issues/419

In the meantime, I don't particularly care to fight against black.  I think I'm just going to merge as is for now.  Hopefully once black releases again this will go back.

Reply:
Ok, to be clear wasn't meaning this comment to hold things up. Just that stuck out to me was all.